backend/tests/yeomanui.spec.ts

```diff
@@ -330,29 +398,34 @@ describe('yeomanui unit test', () => {
 			expect(test3Choice.name).to.be.equal("3rd - Test");
 		});
 
-		it("get generators with homepage", async () => {
```

Comment:
don't delete this test! just don't change it!

Reply:
This test doesn't test anything new, everything it has tested has already been tested in other tests

Reply:
this is unnecessary test

Reply:
Ok. Then revert my last commit